setup.py

```diff
@@ -30,6 +30,7 @@
     'ray',
     'scikit-image',
     'scipy',
+    'setproctitle==1.1.10',
```

Comment:
this is from ray, right? can you leave a comment to that effect

Reply:
It's actually not, it's just used by multiprocessing workers to label themselves as workers.

Reply:
ah -- is there a less-strict constraint we can add? <=, >=, etc?

Reply:
Yeah, I've changed it to just `>=1.0`.